exoctk/contam_visibility/resolve.py

```diff
@@ -1,8 +1,33 @@
+import numpy as np
+from astropy.coordinates import SkyCoord
 from exoctk.utils import get_target_data
 
 def resolve_target(targetName):
-    data = get_target_data(targetName)
-    ra = data['RA']
-    dec = data['DEC']
+    """
+    Gets the RA and Dec of your target.
 
-    return ra, dec
+    Parameters
+    ----------
+    targetName : str
+        The name of your target.
+
+    Returns
+    -------
+    ra : str
+        The Right Ascension (RA) of your target in HH:MM:SS.
+    dec : str
+        The Declination (Dec) of your target in HH:MM:SS.
+    """
+    data, url = get_target_data(targetName)
+    ra_deg = data['RA']
+    dec_deg = data['DEC']
+
+    sc = SkyCoord(ra_deg, dec_deg, unit='deg')
+    ra_hms = '{}:{}:{}'.format(int(sc.ra.hms[0]), \
+                               int(sc.ra.hms[1]), \
+                               np.round(sc.ra.hms[2], 2))
+    dec_hms = '{}:{}:{}'.format(int(sc.dec.dms[0]), \
+                                int(sc.dec.dms[1]), \
+                                np.round(sc.dec.dms[2], 2))

```

Comment:
@jaymedina I thought this was the part you found a one line solution for?

Reply:
@julesfowler nay, I was actually just using that function for testing. the short solution is in `app_exoctk.py`

Reply:
Okay. If this was just used for testing, then let's not keep extra code in our final product. Can you zap what isn't being used?